Dev

nitharsan7 opened the pull request.

markolaban committed:
[RN-94] Make initial setup usable

markolaban committed:
[RN-94] Make initial setup usable

Sam-Boyes-l33t committed:
[RN-94] Formatting error fix

non-default variables cannot follow default variables.

markolaban committed:
[RN-94] Make initial setup usable

Sam-Boyes-l33t committed:
[RN-64] Update based on realscape

root account is now named public in .env for compatibility with realscape.

Sam-Boyes-l33t committed:
[RN-64] Update based on realscape

Add redirect URI to the .env example for local angular development.

Sam-Boyes-l33t committed:
[RS-11] Fix location query

Location based queries used a hardcoded radius of 100 meters. Fixed it to use the query radius, and default to 100 if radius is not included.

markolaban committed:
Adding notes for building from sources on MacOS

markolaban committed:
Merge remote-tracking branch 'origin/dev' into dev

nitharsan7 committed:
[RN-102] bump version

nitharsan7 committed:
Merge branch 'dev' of https://github.com/virtual-space/realnet-server into dev

markolaban committed:
[RN-63] API changes for iOS & Android

markolaban committed:
[RN-63] API changes for iOS & Android

nitharsan7 committed:
[RN-81] auto increase version

nitharsan7 committed:
Merge branch 'dev' of https://github.com/virtual-space/realnet-server into dev

nitharsan7 committed:
bumped_version

nitharsan7 committed:
[RN-81] auto increase version

Sam-Boyes-l33t committed:
[RN-104] Fix location queries

lng/lat was in the wrong order.
Some code cleaning up.

Sam-Boyes-l33t committed:
[RN-105] Update to fix integrity errors.

markolaban committed:
[RS-8] Switch iOS to use new api

markolaban committed:
Merge remote-tracking branch 'origin/dev' into dev

Sam-Boyes-l33t commented:
Looks good, the gitguardian security check is not a problem in this case, but we should add the mob_client_secret as an .env variable.

Sam-Boyes-l33t committed:
[RN-105] allow preset-secrets

same default as before - gen_salt(48)

nitharsan7 committed:
enable CloudWatch Log

nitharsan7 committed:
enable CloudWatch Log

Sam-Boyes-l33t committed:
[RS-43] Backend change to map data

Made it so that backend can handle both point and polygon input data for items. This shouldn't break the currently existing functionality.

Sam-Boyes-l33t committed:
[RS-43] Update backend to allow deleting locations.

markolaban committed:
[RS-72] json type  import works

markolaban committed:
[RS-72] WIP import json types

markolaban committed:
[RS-72] Fixed my items query and updated basic types

markolaban committed:
[RS-72] Implemented csv item import

markolaban committed:
[RS-72] Types now initialized from json

markolaban committed:
Merge branch 'apps-implementation' into dev

markolaban committed:
force merge

markolaban committed:
[RS-72] WIP adding DT UI
